x-pack/test/api_integration/apis/xpack_main/stats/stats.js

```diff
@@ -7,49 +7,39 @@
 import expect from 'expect.js';
 
 export default function ({ getService }) {
-  const supertest = getService('supertest');
   const esArchiver = getService('esArchiver');
+  const statsAPI = getService('statsAPI');
 
   describe('stats API', () => {
     before('load archives', async () => {
-      // Using this archive because it includes reports as well as a range of visualization types.
-      await esArchiver.load('reporting/6_2');
-      // Not really neccessary to have data indexes, but it feels incomplete to leave out, and it is possible that
-      // having data available could potentially interefere with the stats api (unlikely... but possible).
-      await esArchiver.load('logstash_functional');
```

Comment:
don't know who originally wrote that comment (could have been me and I forgot, hah) but it seems like it would be a bug if the stats api relied on data, so I decided to remove this archive unless we have a more concrete reason to pull it back in.